admin/server/users.go

```diff
@@ -321,6 +332,48 @@ func (s *Server) RevokeUserAuthToken(ctx context.Context, req *adminv1.RevokeUse
 	return &adminv1.RevokeUserAuthTokenResponse{}, nil
 }
 
+// findUserAuthTokenFuzzy attempts to find a user auth token by exact ID, full token string, or unique prefix.
+func (s *Server) findUserAuthTokenFuzzy(ctx context.Context, input string) (*database.UserAuthToken, error) {
```

Comment:
Internal functions come after exported functions: https://github.com/uber-go/guide/blob/master/style.md#function-grouping-and-ordering